src/components/CardModal/CardModalContainer.tsx

```diff
@@ -21,6 +22,7 @@ interface PropsFromDispatch {
 
     updateCard: (card: Partial<ICard>) => void
     deleteCard: () => void
+    addCheckList: () => void
```

Comment:
Maybe add the `removeCheckList` for later

Reply:
I prefer to leave it for the person who'll do it. I am not going to add this function without defined it and make all the use case process in the same time. I can do it, but I am on redux tests.